docs/architecture/blazor-for-web-forms-developers/introduction.md

```diff
@@ -8,6 +8,50 @@ ms.date: 09/11/2019
 
 # An introduction to Blazor for ASP.NET Web Forms developers
 
-ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. 
+ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. Back when the Web was still largely in its infancy, ASP.NET Web Forms made building web apps simple and productive by adopting many of the patterns that were used for desktop development. In ASP.NET Web Forms apps, web pages are composed from reusable UI controls. User interactions are handled as events. A rich ecosystem of Web Forms UI controls provided by Microsoft and third-party control vendors makes connecting to data and displaying rich visualizations easy. For the visually inclined, the Web Forms designer provides a simple drag-and-drop interface for laying out controls.
 
+Over the years, Microsoft has introduced new ASP.NET-based web frameworks to address new trends in web development, such as ASP.NET MVC, ASP.NET Web Pages, and more recently ASP.NET Core. With each new framework, some have predicted the imminent decline of ASP.NET Web Forms and criticized it as an outdated and outmoded web framework. Despite these predictions, many .NET web developers continue to find ASP.NET Web Forms a simple, stable, and productive way to get their work done. At the time of this writing, almost half a million web developers use ASP.NET Web Forms every month. ASP.NET Web Forms is stable to the point that docs, samples, books, and blog posts from a decade ago remain useful and relevant. For many .NET web developers, "ASP.NET" is still virtually synonymous with "ASP.NET Web Forms" as it was when .NET was first conceived. Arguments may rage on the pros and cons of ASP.NET Web Forms compared to the other new .NET web frameworks, but ASP.NET Web Forms development remains a popular way to write web apps.
```

Comment:
I think we should be careful about what we identify as a framework, as there's a significant difference in granularity. I suggest using a term like "application model" to categorize app types (ASP.NET MVC and ASP.NET Web Pages) within ASP.NET 4.x. Use the term "framework" to categorize ASP.NET Core. Maybe something like the following:

```suggestion
Over the years, Microsoft has introduced new ASP.NET-based application models to address new trends in web development, such as ASP.NET MVC and ASP.NET Web Pages. More recently, ASP.NET Core was released as a reimagining of the ASP.NET framework. With each new application model and framework, some have predicted the imminent decline of ASP.NET Web Forms and criticized it as an outdated and outmoded web framework. Despite these predictions, many .NET web developers continue to find ASP.NET Web Forms a simple, stable, and productive way to get their work done. At the time of this writing, almost half a million web developers use ASP.NET Web Forms every month. ASP.NET Web Forms is stable to the point that docs, samples, books, and blog posts from a decade ago remain useful and relevant. For many .NET web developers, "ASP.NET" is still virtually synonymous with "ASP.NET Web Forms" as it was when .NET was first conceived. Arguments may rage on the pros and cons of ASP.NET Web Forms compared to the other new ASP.NET application models, but ASP.NET Web Forms development remains a popular way to write web apps.
```

Reply:
I'd rather not try to make this distinction between frameworks and app models. I don't think the difference is as well defined as this. Frameworks come in all shapes and sizes :smile:.